ui/DockerfileDev

```diff
@@ -1,8 +1,9 @@
-FROM node:22-alpine as build
+FROM node:22-alpine AS build
 
 ENV NODE_ENV=development
 
-RUN apk add --no-cache git bash
```

Comment:
Es buildet bei mir mittlerweise mehr als 10 Minuten in dem Schritt. Wie lange soll ich warten? :D

Reply:
`npm update -g npm` benötigt sehr lange, dafür sollte es das Problem lösen, dass der `ui` Container beim Starten hängen bleibt: https://github.com/nodejs/docker-node/issues/1946#issuecomment-2492456441
e